packages/apps/job-launcher/server/src/modules/auth/auth.controller.ts

```diff
@@ -3,84 +3,83 @@ import {
   ClassSerializerInterceptor,
```

Comment:
Can you explain, why did you remove `refreshToken` logic?

Reply:
It has not been removed. The refresh token has been included inside the jwt. As far as I understood the refresh token was used to get a new jwt and to set the jwt expired (logout). For those two actions you need to authenticate the user through the jwt, so it can be included in it